Configuration/DataProcessing/python/Repack.py

```diff
@@ -55,7 +55,8 @@ def repackProcess(**args):
 
         outputModule = cms.OutputModule(
             "PoolOutputModule",
-            RAWEventContent,
+            compressionAlgorithm=RAWEventContent.compressionAlgorithm,
+            compressionLevel=RAWEventContent.compressionLevel,
```

Comment:
In general it would be better to do
```suggestion
            compressionAlgorithm=cms.untracked.string(RAWEventContent.compressionAlgorithm.value()),
            compressionLevel=cms.untracked.int32(RAWEventContent.compressionLevel.value()),
```

Otherwise if anything changes `RAWEventContent.compressionAlgorithm` or `RAWEventContent.compressionLevel` after these lines (probably unlikely to happen on purpose though), those changes will propagate here as well, which can be confusing. Explicitly copying the values instead of using the same python objects prevents such "long distance interactions".

Reply:
I would instead argue that if RAWEventContent.compressionLevel changes type, then it should propagate here as well as it means the type of that argument to the output module class has also changed. (so I don't agree that this change is an improvement)

Reply:
Point taken. One could do also
```suggestion
            compressionAlgorithm=copy.copy(RAWEventContent.compressionAlgorithm),
            compressionLevel=copy.copy(RAWEventContent.compressionLevel),
```
(which then needs `import copy` somewhere here) that would satisfy both goals.

Reply:
ah - sorry, I had misunderstood... I think this sort of use-without-changing is routinely done, but indeed if the repack script is later changed to alter these parameters, then this is safer.